apps/omg_watcher/test/web/controllers/transaction_test.exs

```diff
@@ -111,6 +111,42 @@ defmodule OMG.Watcher.Web.Controller.TransactionTest do
     end
   end
 
+  describe "Controller.TransactionTest - transactions" do
+    @tag fixtures: [:initial_blocks]
+    test "endpoint returns last 3 transactions", %{
+      initial_blocks: _initial_blocks
+    } do
+      limit = 3
+      %{"data" => txs, "result" => "success"} = TestHelper.rest_call(:get, "/transactions?limit=#{limit}")
+
+      assert [
+               %{"txblknum" => 3000, "txindex" => 1},
+               %{"txblknum" => 3000, "txindex" => 0},
+               %{"txblknum" => 2000, "txindex" => 0}
+             ] == Enum.map(txs, fn tx -> %{"txblknum" => tx["txblknum"], "txindex" => tx["txindex"]} end)
+    end
+
+    @tag fixtures: [:initial_blocks, :alice]
+    test "endpoint returns last 2 transactions filtered by address", %{
+      initial_blocks: _initial_blocks,
+      alice: alice
+    } do
+      limit = 2
+      {:ok, address_encoded} = Crypto.encode_address(alice.addr)
+      address = alice.addr |> TestHelper.to_response_address()
+
+      %{"data" => txs, "result" => "success"} =
+        TestHelper.rest_call(:get, "/transactions?address=#{address_encoded}&limit=#{limit}")
+
+      assert [
+               %{"txblknum" => 3000, "txindex" => 1},
+               %{"txblknum" => 3000, "txindex" => 0}
+             ] == Enum.map(txs, fn tx -> %{"txblknum" => tx["txblknum"], "txindex" => tx["txindex"]} end)
+
+      assert [true, true] == for(tx <- txs, do: tx["newowner1"] == address or tx["newowner2"] == address)
```

Comment:
Please add more test
- [ ] shows tx that contain requested address as the `sender` and not `recipient`
- [ ] tx contains requested address as both sender & recipient is listed once
- [ ] tx without inputs filtered by recipient
- [ ] tx without outputs (`amount = 0`) filtered by sender
- [ ] queries on empty DB (both with/out limit, address) returns empty array
Please try to be explicit in assertions, e.g. expected is recipient 1 to have specified address